### DISCUSSION TITLE
Discord unban request

### DISCUSSION THREAD

**Discussion by skretch92** [chromiecraft/chromiecraft#3302] [General]
Hi,
I'm writing here since this is the only way to contact server staff (I hope). Some time ago I got banned from discord server 'cause my account has been hacked and i started spamming (I suppose). It seems to be an IP ban because I can't rejoin server even with different accounts.
My question is: is there a way to join server again? I need to open a ticket and to do that I need to use discord.

Thank you in advance.

**Comment by rerongwe** [comment]
well, i have exactly the same problem and dont know too what to do.

**Comment by 55Honey** [comment]
Hi @skretch92 @rerongwe 
Please remind me of your discord IDs ?

**Reply by skretch92** [reply]
Hi @55Honey, this is my ID **124893552860725250**. Let me know if this is what you need or you need something else!

Thank you.

**Reply by 55Honey** [reply]
@skretch92 are you sure your account is not compromised anymore? What steps did you take to make it secure?

**Reply by skretch92** [reply]
Well, of course i changed password with a very different and stronger one. 2FA was already active since it's a requirement to join the server.
After password change I immediatly stopped to send spam messages, so I'm pretty sure that I'm not compromised anymore.

**Reply by 55Honey** [reply]
Your account is unlocked on our discord. Be careful with clicking links ;)
Welcome back.

**Reply by skretch92** [reply]
Thank you so much, have a good day!

**Comment by rerongwe** [comment]
Hi @55Honey, my user ID is 874802675034755103, username rerong#0234
Thnks

**Reply by 55Honey** [reply]
@rerongwe This discord id is not banned on the ChromieCraft discord. You can join anytime on https://discord.gg/chromiecraft

**Reply by rerongwe** [reply]
![image](https://user-images.githubusercontent.com/103150948/162573594-0d5ef354-42b9-47d2-abdc-a0593e869b18.png)

this is the msg error when i try to join server, and other servers this dont show up

**Reply by rerongwe** [reply]
i already have phone number register, 2FA, so dont know why this keep show up

**Reply by 55Honey** [reply]
I have double-checked the ID 874802675034755103 as well as your name rerong#0234.
Both do not show up in our list of banned members. I suggest you ask the discord-support about this.

**Reply by rerongwe** [reply]
okay, can u check the ID 874802675034755103 too? this one is from discord mobile dont know if helps u, i create another account of discord and looks like fix my problem, but will be great if i can use my rerong#0234 account

**Reply by rerongwe** [reply]
the new ID is 959801019104129025, sorry *

**Reply by rerongwe** [reply]
Looks like i know what is the problem, is the number verification thing, when i put my number any server won't show up to join, but when i remove my number i can join server

**Reply by rerongwe** [reply]
but i cant talk on chat because need verify phone number
